codes/classical/Cycle Codes.yml

```diff
@@ -0,0 +1,34 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+
+# code id, physical, logical are all lower case
+code_id: cycle
+physical: bits
+logical: bits
+
+
+name: 'cycle code of graphs '
+introduced: '\cite{https://ieeexplore.ieee.org/document/394711}'
```

Comment:
\cite{doi:10.1109/ISIT.1994.394711}
please check other citations

Reply:
These codes were introduced some 30 years before the cited papers in 
https://doi.org/10.1109/TIT.1968.1054190.
Please correct.